docs/robots/ridgeback/integration_ridgeback.mdx

```diff
@@ -1,12 +1,161 @@
+batteries are positioned low in the chassis and slightly rearward of center of the robot to balance the
+weight distribution when mounting front-facing manipulator payloads. To minimize the possibility of tipping
+over, payload structures should always be mounted as close to center as possible.
+
+##### Payload Mounting Holes
+
+Located at the front-end of the mounting plate are two 5/8"-11 screw holes for mounting Baxter, UR5/UR510 manipulator
+arms, or any other payload structure. These holes are indicated in the figure below. If you purchased the Baxter
+or UR5/UR10 package from Clearpath Robotics, Ridgeback will come with the required hardware adapters to securely mount
+the payload structures to the plate.
+
+If your payload structure requires additional mounting holes or a different hole configuration, the plate can be removed
+and additional holes can be drilled. To remove the mounting plate, simply remove the screws that form the diamond pattern
+on top of the plate.
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/ridgeback_images/payload_integration_plate.png"
+      width="600"
+    />
+    <figcaption>Payload integration plate</figcaption>
+  </figure>
+</center>
+
+:::warning
+
+Permanent damage resulting from custom modifications to the mounting plate is not covered under warranty and may not be
+supported by Clearpath Support. Please contact our support team if you require assistance or have any questions relating
+to custom modifications.
+
+:::
+
+### Electrical Integration {#ridgeback-payload-electrical-integration}
+
+Except for bus-powered USB cameras, most payloads have separate leads for power and data.
+
+#### Data Connections
+
+The two larger holes on the left and right side of the payload integration plate allow you to pass
+electrical wires and cables from the mounted payloads into the User Bay. Electrical wires should always
+pass through the provided plastic grommets to protect against cutting and abrasion.
+
+Ridgeback's internal computer options support USB3 and Ethernet connectivity. Where available, the
+PCIe slot may be used to supply Firewire, Thunderbolt, or additional USB3 ports, as necessary.
+
+Additionally, the User Bay may be used for an Ethernet switch, when attaching multiple
+Ethernet payloads, or for a PoE power injector as required.
+
+#### Connector Summary {#ridgeback-connector-summary}
+
+Several user power connectors are accessible in the User Bay for powering user payloads.
+These are summarized in the following table and described in more detail in the following sections.
+The power rails on the User Bay Power board are protected against short circuit by fuses. See the
+illustration for the connector and fuse locations.
+
+- For the High Current VBAT User Power (labelled as 24 VDC, 20A MAX), terminate your payload's power lead with the appropriate Anderson connectors.
+  TODO: why is it a 7.5A fuse in figure??
```

Comment:
You could change to an image from VKS:

https://clearpathrobotics.vkscentral.com/asset/image/295069?hash=b781737398e09d0112fe58897c03966a&created_at=2016-04-07&original_name=img_0180.jpg&mime=image/jpeg&org=65&isStrict=1

Reply:
Updated to a new image with proper fuse values to match what is in VKS.